src/analyzer/analyzerebur128.cpp

```diff
@@ -69,7 +69,13 @@ void AnalyzerEbur128::storeResults(TrackPointer pTrack) {
         qWarning() << "AnalyzerEbur128::storeResults() failed with" << e;
         return;
     }
-    if (averageLufs == -HUGE_VAL || averageLufs == 0.0) {
+    if (averageLufs == -HUGE_VAL ||
+            averageLufs == HUGE_VAL ||
+            // This catches 0, inf and -inf (that may have slipped through in
+            // libebur for some reason.
+            // TODO Move this to a VERIFY_OR_DEBUG_ASSERT() once the root cause
+            // has been identified, and only do a zero check here.
+            util_isnormal(averageLufs)) {
```

Comment:
Now the check for 0 is gone. 
My idea was to just

```suggestion
    if (!util_isnormal(averageLufs) || averageLufs == 0.0) {
            // This catches 0, inf (HUGE_VAL), -inf (-HUGE_VAL) and denormals (treated as zero), 
            // that may have slipped through in libebur for some reason.
```

Returning -HUGE_VAL is the documented invalid value. I am in doubt there will be a "fix" for HUGE_VAL and even then we deal with user data, where Bad things are not "impossible". So I would not add the TODO.

Reply:
Zero is covered by std::isnormal
https://en.cppreference.com/w/cpp/numeric/math/isnormal

I'll remove the TODO then.

Reply:
Ups, thank you for the hint.